src/lumigo_opentelemetry/instrumentations/grpcio/grpc_instrument_server.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Any
+
+from opentelemetry.instrumentation.grpc._server import OpenTelemetryServerInterceptor
+from opentelemetry.trace import get_current_span
+from grpc._utilities import RpcMethodHandler
+from grpc._server import _RequestIterator
+
+from .config import PAYLOAD_MAX_SIZE
+
+
+def request_wrapper(request: _RequestIterator) -> None:
+    history: List[str] = []
+    original_next = request._next
+
+    def wrapped_next() -> Any:
+        data_chunk = original_next()
```

Comment:
isn't this the same logic as the `iterator_wrapper`? can we move it to a common module?